.github/workflows/unity-nuget-test.yml

```
@@ -8,13 +8,81 @@ permissions:
   contents: read
 
```

```suggestion
concurrency:
  group: unity-test
  cancel-in-progress: false

```


https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs